src/renderer/views/RegisterView/subviews/ActivationWaitSubview.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from "react";
+import { ACTIVATION_DOCUMENTATION_LINK } from "src/config";
+import H1 from "src/renderer/components/ui/H1";
+import { ExtLink } from "src/renderer/components/ui/Link";
+import { useActivate } from "src/utils/useActivate";
+import { useActivationStatus } from "src/utils/useActivationStatus";
+import { RegisterState } from "..";
+
+interface Props {
+  setState: (state: RegisterState) => void;
+}
+
+function ActivationWaitSubview({ setState }: Props) {
+  const activate = useActivate();
+  const { activated, error } = useActivationStatus();
+
+  useEffect(() => void activate(), [activate]);
```

Comment:
Need error handling according to error types of `useActivate()`, 
Recommend define and export error types in `src/interfaces` for better modularity.

also we would need state definition on failure of this hook.

Reply:
- [x] Handle error from `useActivate()`
- [ ] Move error types to `src/interfaces`